java/build-nnstreamer-android.sh

```diff
@@ -122,9 +122,9 @@ flatbuf_ver="1.12.0"
 enable_mqtt="no"
 paho_mqtt_c_ver="1.3.7"
 
-# Set tensorflow-lite version (available: 1.9.0 / 1.13.1 / 1.15.2 / 2.3.0)
-tf_lite_ver="2.3.0"
-tf_lite_vers_support="1.9.0 1.13.1 1.15.2 2.3.0"
+# Set tensorflow-lite version (available: 1.9.0 / 1.13.1 / 1.15.2 / 2.3.0 / 2.7.0)
+tf_lite_ver="2.7.0"
```

Comment:
You should fix line 519 as default version in .mk file is changed.
(sed -i "s|TFLITE_VERSION := 2.3.0|...)